### PR TITLE
Update PAO subscription channel and bump dependencies

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: cnf-tests/TESTLIST.md [hunk 1]
```diff
@@ -90,26 +90,27 @@ The cnf tests instrument each different feature required by CNF. Following, a de
 
 | Test Name | Description |
```
Comment (fromanirh) [inline cnf-tests/TESTLIST.md:91]: are these changes relevant to this patch?

File: cnf-tests/docgen/e2e.json [hunk 2]
```diff
@@ -14,11 +14,12 @@
     "[ptp] PTP e2e tests PTP Interfaces discovery The virtual interfaces should be not discovered by ptp": "Checks that the virtual interfaces are not used by the ptp daemon",
     "[ptp] PTP e2e tests PTP metric is present on slave": "Checks that the metrics related to ptp are produced by the slave.",
     "[rfe_id:27350][performance]Topology Manager [test_id:26932][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] should be enabled with the policy specified in profile": "Verifies that when specifying a topology policy in the profile, that is used by the topology manager.",
-    "[rfe_id:27363][performance] CPU Management Verification of configuration on the worker node [test_id:28026][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify CPU affinity mask, CPU reservation and CPU isolation on worker node": "Verifies that CPU affinity, reservation and isolation are set correctly on the node as specified in the profile spec.",
```
Comment (fromanirh) [inline cnf-tests/docgen/e2e.json:17]: same

**Review by fromanirh — COMMENTED** [review]
changes per se looks good but something extra sneaked in?

File: cnf-tests/TESTLIST.md (hunk 1 above)
Comment (cynepco3hahue) [reply cnf-tests/TESTLIST.md:91]: I needed the vendor update because RT kernel fix, but it brought some new tests, and CI requires that each test will have a description under the document, so I needed to add it

File: cnf-tests/docgen/e2e.json (hunk 2 above)
Comment (cynepco3hahue) [reply cnf-tests/docgen/e2e.json:17]: same as above

File: cnf-tests/TESTLIST.md (hunk 1 above)
Comment (fromanirh) [reply cnf-tests/TESTLIST.md:91]: Fair enough. Good enough for me. Let's hear from @fedepaol to make sure the network squad is OK with that.

**Comment by cynepco3hahue** [comment]
@fromanirh @fedepaol Hi folks, can I get a review for this one?

**Comment by fromanirh** [comment]
/hold
/lgtm
from my side seems good. I'll leave the final call to @fedepaol , if he's happy we can remove the hold.

File: cnf-tests/TESTLIST.md (hunk 1 above)
Comment (fedepaol) [reply cnf-tests/TESTLIST.md:91]: Cool, it means that it really works :+1: 
Yes, the idea is that whenever we add new tests, we provide a valid description and this is a way not to forget about it.

**Comment by fedepaol** [comment]
I am totally happy :clown_face: 
/hold cancel
/approve
